sdv/logging/sdv_logger_config.yml

```diff
@@ -6,22 +6,22 @@ loggers:
     propagate: false
     handlers:
       class: logging.FileHandler
```

Comment:
instead of `logging.FileHandler` this should be `sdv.logging.logger.CSVHandler`

Reply:
I think here the CSV part is being handled at the Formatter level, not the handler level.

Reply:
Yeah sorry the `CSVHandler` threw me off thinking as a `CSVHandler` like inheriting from `FileHandler`